src/objects.jl

```diff
@@ -173,7 +137,9 @@ end
 
 type SDDPparameters
     # Solver used to solve LP
-    solver
+    LPSOLVER::MathProgBase.AbstractMathProgSolver
```

Comment:
Maybe simply differentiate MILP solver and Solver for the moment if it is useful. I think most people will use the same solver in both case.